internal.d.ts

```diff
@@ -156,12 +122,12 @@ interface PortalProps {
 }
 
 type RoactEvents<T> = {
-	[K in keyof Partial<SubType<T, RBXScriptSignal>>]: T[K] extends RBXScriptSignal<infer F>
+	[K in keyof SubType<T, RBXScriptSignal>]: T[K] extends RBXScriptSignal<infer F>
 		? EventHandlerFunction<T, FunctionArguments<F>>
 		: never
 };
 
-type RoactPropertyChanges<T> = { [key in keyof Partial<SubType<T, PropertyTypes>>]: PropertyChangeHandlerFunction<T> };
+type RoactPropertyChanges<T extends Instance> = { [key in keyof GetWritableProperties<T>]: (rbx: T) => void };
```

Comment:
The current implementation, `Change` is broken due to matching the property key's props rather than the actual properties.

This change fixes it:
```ts
type RoactPropertyChanges<T extends Instance> = {
	[key in GetWritableProperties<T>]?: (rbx: T) => void
};
```

(basically remove `keyof`, make it partial)

Reply:
Good catch. Thanks for reviewing this!